jenkins/Jenkinsfile

```diff
@@ -10,6 +10,21 @@ def vEnv(){
       virtualenv .venv
     ''', label: "creating virtualenv"
 }
+
+def doGetVersion(){
+  sh(script: '''#!/bin/sh -e
+    . cloudify-manager/packaging/version_info
+    echo ${CLOUDIFY_VERSION}
+  ''', label: 'get package version', returnStdout: true).trim()
+}
+
+def doGetPreRelease(){
```

Comment:
Why not just get${thing}?

Reply:
I'd like to avoid name conflicts between this and the pipeline-shared-library

Reply:
After discussion externally: Namespacing fun.